package.json

```diff
@@ -29,6 +29,7 @@
     "copy-notices": "node scripts/copy-notices.js",
     "create-component": "plop --plopfile ./scripts/create-component/create-component.ts --dest . --require ./scripts/ts-node-register",
     "create-package": "plop --plopfile ./scripts/create-package/plopfile.ts --dest . --require ./scripts/ts-node-register",
+    "e2e": "lage e2e --reset-cache --verbose --mode run",
```

Comment:
_Curious:_ what does `--reset-cache` there and why it's required?

Reply:
Removed it, it was because i wanted to test the script a few times and `Lage` wouldn't run because of the cache. Nice catch